src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -73,12 +69,9 @@ export const SearchResults = ({
   });
 
   useEffect(() => {
-    if (doesCookieExist()) {
-      setModalState({
-        ...deriveDefaultModalState(),
-        doesCookieExist: doesCookieExist(),
-      });
-    }
+    setModalState({
+      ...deriveDefaultModalState(),
+    });
   }, []);
```

Comment:
I know this was merged already, but this useEffect appears to no longer be necessary.

Reply:
My understanding was that this hook was needed to establish a default values for the modal state. But I didn't test that; I'll have a deeper look.

Reply:
The default value is set on line 60 with the `useState` call. The `useEffect` _was_ necessary because things were getting (conditionally) changed from the default value if the cookie existed. The fact that the `useEffect` took no action in the event that `doesCookieExist()` returns `false` supports the idea that it can now be removed altogether.

Reply:
@ryguyk makes perfect sense to me now that you point it out. I'll get a PR up shortly to remove the cruft.

Reply:
PR to remove the unnecessary useEffect hook: https://github.com/department-of-veterans-affairs/vets-website/pull/22523